sdk/messaging/src/main/java/com/klaviyo/messaging/KlaviyoWebView.kt

```diff
@@ -56,38 +39,18 @@ class KlaviyoWebView : WebViewClient(), WebViewCompat.WebMessageListener {
         it.settings.userAgentString = DeviceProperties.userAgent
         it.settings.javaScriptEnabled = true
 
-        if (USE_NEW_FEATURES && isFeatureSupported(WEB_MESSAGE_LISTENER)) {
+        if (isFeatureSupported(WEB_MESSAGE_LISTENER)) {
             Registry.log.verbose("$WEB_MESSAGE_LISTENER Supported")
             WebViewCompat.addWebMessageListener(
                 it,
                 JS_BRIDGE_NAME,
-                setOf(Registry.config.baseUrl),
+                setOf("http://a.local-klaviyo.com:8080"),
```

Comment:
```suggestion
                //TODO - sort out how to toggle between local and production. 
                setOf("http://a.local-klaviyo.com:8080"),
```
For viz to any other reviewers, we're temporarily just pointing to local to reduce confusion, plan to sort out how to toggle this from the test app shortly.